.github/workflows/slow-tests.yml

```diff
@@ -0,0 +1,17 @@
+name: slow-tests
+on: [workflow_dispatch]
```

Comment:
Example of a run manually triggered in my fork: https://github.com/nshalman/fsnotify/actions/runs/1708324643